packages/flutter_tools/lib/src/application_package.dart

```diff
@@ -166,8 +166,11 @@ class AndroidApk extends ApplicationPackage {
 
     final File manifest = androidProject.appManifestFile;
 
-    if (!manifest.existsSync())
+    if (!manifest.existsSync()) {
+      printError('AndroidManifest.xml could not be found.');
+      printError('Please check ${manifest?.path ?? '??'} for errors.');
```

Comment:
Can we tell them where we expect to find this if it's not there?  e.g. `?? 'android/app/src/main/AndroidManifest.xml`?